packages/labs/gen-wrapper-vue/src/lib/package-json-template.ts

```diff
@@ -42,7 +42,7 @@ export const packageJsonTemplate = (
         '@vitejs/plugin-vue': '^3.1.2',
```

Comment:
Non blocking: The generator sets TypeScript to `~4.7.4` as default. Should this also be bumped to TS 5.2?

Reply:
Yeah, I'll bump that back up. We'll only have test coverage of one version at a time, so it should match what we test.